### ISSUE TITLE
"simple" vector field plot not displaying many points

### ISSUE THREAD

**Issue by sritchie** [stevenpetryk/mafs#164]
Hi @stevenpetryk! I was playing around with some vector field examples, and noticed that some simple functions,  like `([x, y]) => [x, y]` or similar versions with `[y, x]`, `[y, -x]` etc, all failed to fill in values in 3 quadrants:

<img width="572" alt="image" src="https://github.com/stevenpetryk/mafs/assets/69635/aac17577-f2a2-4919-a77f-2b936b93100f">

Here's a stackblitz reproducing the problem:

https://stackblitz.com/edit/react-aqgdx6?file=src%2Findex.tsx

and the code:

```js
import * as React from 'react';
import { createRoot } from 'react-dom/client';

import 'mafs/core.css';
import 'mafs/font.css';

import { Mafs, Plot, Coordinates, useMovablePoint } from 'mafs';

function VectorFieldExample() {
  return (
    <Mafs>
      <Coordinates.Cartesian subdivisions={2} />
      <Plot.VectorField
        xy={([x, y]) => [y, -x]}
        step={0.25}
        xyOpacity={([x, y]) => 0.5}
      />
    </Mafs>
  );
}

createRoot(document.getElementById('root')).render(<VectorFieldExample />);
```

If I change the step size to a non-multiple-of-0.25, like `0.25001`, then 2 more quadrants appear but the top right is missing:

<img width="594" alt="image" src="https://github.com/stevenpetryk/mafs/assets/69635/f7cf0625-502f-406b-9ef7-fde9cb133e6d">

I don't see anything in the Stackblitz logs. Apologies if there is something obvious I'm not seeing in this example!

**Comment by stevenpetryk** [comment]
Looks like some NaNs are ending up inside the SVG path:
![](https://github.com/stevenpetryk/mafs/assets/1724000/54019ae3-baf4-4e4e-9d5c-b338c44d0e99)

**Comment by stevenpetryk** [comment]
[Fixed in v0.18.8](https://github.com/stevenpetryk/mafs/compare/v0.18.7...v0.18.8#diff-f7c86ae71b7b10ecbbde5881405eea3e5bcf834e7048a2dd533d42d4e01fbd39).

**Comment by sritchie** [comment]
Thanks @stevenpetryk !!
